.github/workflows/minimum_code_coverage.yaml

```
@@ -31,4 +31,4 @@ jobs:
 
     - name: Check code coverage is 100%
       run: |
-        pytest --cov=./blueprints --cov-fail-under=100
\ No newline at end of file
+        pytest --cov=./blueprints --cov-report term-missing --cov-fail-under=100
```

according to [this documentation](https://pytest-cov.readthedocs.io/en/latest/reporting.html) it is possible to skip the 100% covered lines by --cov-report term-missing:skip-covered
I think it would be less messy. Let me know if you disagree.

```suggestion
        pytest --cov=./blueprints --cov-report term-missing:skip-covered --cov-fail-under=100
```